cpp/mail_krimdok_newsletter.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+
+BACKUP_DIR=/usr/local/var/lib/tuelib/sent_newsletters
+mkdir --parents "$BACKUP_DIR"
+REPLY_TO=johannes.ruscheinski@uni-tuebingen.de
```

Comment:
Don't forget to change the hardcoded reply later

Reply:
Yup!